editor/src/clj/editor/gui.clj

```diff
@@ -3960,17 +3958,11 @@
       nil)))
 
 (defn- handle-drop
-  [action op-seq]
-  (let [{:keys [string gesture-target]} action
-        ui-context (first (ui/node-contexts gesture-target false))
-        {:keys [selection workspace]} (:env ui-context)
-        resources (->> (str/split-lines string)
-                       (keep (partial workspace/resolve-workspace-resource workspace)))]
```

Comment:
It looks like we used to support dropping text from an external application onto it here, and creating resources from newline-separated paths. Presumably that no longer works?

I wasn't aware of such a feature before, and it seems a bit strange to only support it in Guis. I guess we could wait until users complain and re-implement it if they do.

Reply:
That was introduced here https://github.com/defold/defold/pull/10461#discussion_r2035526622, to avoid handling dragged assets as files.

Reply:
It does block the feature you describe, so it's a questionable decision. Since we didn't have a similar request, I decided to use plain strings.